.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,43 @@
+# Github action definitions for unit-tests with PRs.
+
+name: tfma-unit-tests
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+  workflow_dispatch:
+
+jobs:
+  unit-tests:
+    if: github.actor != 'copybara-service[bot]'
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: ['3.9', '3.10']
```

Comment:
Looking at `README.md` looks like `3.11` is also supported?

Reply:
I vaguely remember an issue with 3.11. I just added it, so we'll see what happens

Reply:
[There appears to be a dependencies issue with 3.11](https://github.com/tensorflow/model-analysis/actions/runs/11475970116/job/31934965212?pr=189).

Reply:
I am guessing probably because 3.11 would require different version of `protobuf-compiler`, similar to the different versions of library specified here: https://github.com/tensorflow/model-analysis/blob/7a068ec4e15224f5a1e3f23a7bcbd9bd4b030181/setup.py#L308

but, either ways it's okay to skip 3.11 in this PR and can be handled later.

Reply:
> it's okay to skip 3.11 in this PR and can be handled later.

I'd rather do this than try to make 3.11 work in this PR